composer.json

```diff
@@ -78,7 +78,8 @@
     "post-install-cmd": [
       "@php artisan clear-compiled",
       "@php artisan cache:clear",
-      "@php -r \"if (!file_exists('.env')) copy('.env.example', '.env');\""
+      "@php -r \"if (!file_exists('.env')) copy('.env.example', '.env');\"",
+      "@php -r \"copy('.htaccess.example', './public/.htaccess');\""
```

Comment:
Shouldn't we check if `./public/.htaccess` exists before copying, just like the case with `.env`?

Reply:
May be we should rename this PR because this is not about just setting directives for file size. It gives flexibility to modify the Apache config.

Reply:
> There is already [.htaccess](https://github.com/koel/koel/blob/master/public/.htaccess?rgh-link-date=2024-04-24T04%3A33%3A40Z) there, if we put the checking then I think .htaccess.example will never get copied.

The current `.htaccess` file should be removed.